schemes/zhang_mcfarlane/zm_prepare_flux_for_check_energy.meta

```diff
@@ -0,0 +1,67 @@
+[ccpp-table-properties]
+  name = zm_prepare_flux_for_check_energy
+  type = scheme
+
+[ccpp-arg-table]
+  name  = zm_prepare_flux_for_check_energy_run
+  type  = scheme
+[ ncol ]
+  standard_name = horizontal_loop_extent
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ prec_dp ]
+  standard_name = lwe_precipitation_rate_at_surface_due_to_deep_convection
+  units = m s-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent)
+  intent = in
+[ rliq ]
+  standard_name = vertically_integrated_cloud_liquid_water_tendency_due_to_all_convection_to_be_applied_later_in_time_loop
+  units = m s-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent)
+  intent = in
+[ snow_dp ]
+  standard_name = lwe_frozen_precipitation_rate_at_surface_due_to_deep_convection
+  units = m s-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent)
+  intent = in
+[ rice ]
+  standard_name = vertically_integrated_cloud_ice_tendency_due_to_all_convection_to_be_applied_later_in_time_loop
```

Comment:
We don't have to do it for this PR, but I just realized that we probably need to specify that this is liquid water equivalent cloud ice mass.  Maybe worth opening an issue?

```suggestion
  standard_name = vertically_integrated_lwe_cloud_ice_tendency_due_to_all_convection_to_be_applied_later_in_time_loop
```